libbpfgo.go

```diff
@@ -341,6 +470,30 @@ func (m *Module) GetMap(mapName string) (*BPFMap, error) {
 	}, nil
 }
 
+func (b *BPFMap) Name() string {
+	cs := C.bpf_map__name(b.bpfMap)
```

Comment:
Quoting from CreateMap above:

> // and therefore do not have access to libbpf high level bpf_map__* APIS

So, this won't work for maps created with CreateMap, right?